### PR TITLE
[3638377425] Feature/update readme

### PR DESCRIPTION
**Pull request by HarrisonHough**
## Description

-   First pass on ReadMe for Core module 

#### Updated

-   ReadMe.md

## How to Test

-   Add steps to locally test these changes

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.
-   [ ] QA Testing is updated.

## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
-# Unity-Core
\ No newline at end of file
+# Unity-Core
```
Comment (srcnalt) [inline README.md:1]: Can we name it `Ready Player Me Unity SDK Core` here? I was thinking in general if we should stuck SDK in the names of the packages and repo names too but I think later modules won't make sense to call SDK maybe. However, we need to make sure people understand at the first glance that this is a Unity-related project. What are your thoughts?

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+Please visit the online documentation and join our public `discord` community.
+
+![](https://i.imgur.com/zGamwPM.png) **[Online Documentation]( https://readyplayer.me/docs )**
+
+![](https://i.imgur.com/FgbNsPN.png) **[Discord Channel]( https://discord.gg/9veRUu2 )**
+
+
+## Quick Start 
+***NOTE: This plugin supports Unity Version 2020.3 or higher***
```
Comment (srcnalt) [inline README.md:13]: Could you also note that they need git installed? And drop the URL of it https://git-scm.com

Comment (HarrisonHough) [reply README.md:13]: ah yes forgot about that 🗡️

Comment (HarrisonHough) [reply README.md]: I added a small requirements section to the start of he quick start 
image

File: README.md (hunk 1 above)
Comment (HarrisonHough) [reply README.md]: Updated

**Review by srcnalt — APPROVED** [review]

**Pull request merged**